src/main/controllers/TribunalNationalListsController.ts

```diff
@@ -31,16 +33,32 @@ export default class TribunalNationalListsController {
 
             const pageLanguage = publicationService.languageToLoadPageIn(metaData.language, req.lng);
 
+            const returnedCourt = await locationService.getLocationById(metaData['locationId']);
+            const courtName = locationService.findCourtName(returnedCourt, req.lng, listToLoad);
+
+            let languageResource = {
+                ...req.i18n.getDataByLanguage(pageLanguage)[listToLoad],
+                ...req.i18n.getDataByLanguage(pageLanguage)['list-template'],
+            };
+
+            if (listToLoad === 'care-standards-list') {
```

Comment:
Could we try putting this at the top of the languageResource block, as hopefully it will get overridden in the other language files where needed

Reply:
care-standards-list specific condition has been removed